### PR TITLE
Added basic support for RemoteProjectEventStream in CompositeView sources

### PR DESCRIPTION
**Pull request by umbreak**
Fixes https://github.com/BlueBrain/nexus/issues/998

### CODE REVIEW

File: src/main/resources/schemas/view.json [hunk 1]
```diff
@@ -234,6 +237,82 @@
...
+          "nodeKind": "sh:IRI"
+        },
+        {
+          "path": "nxv:resourceTag",
+          "name": "Resource tag",
+          "description": "The tag for which indexing will be triggered",
+          "datatype": "xsd:string"
+        },
+        {
+          "path": "nxv:includeMetadata",
```
Comment (bogdanromanx) [inline src/main/resources/schemas/view.json]: is this relevant for a source?

File: src/main/resources/schemas/view.json [hunk 2]
```diff
@@ -234,6 +237,82 @@
...
+          "description": "The project label.",
+          "datatype": "xsd:string",
+          "minCount": 1
+        },
+        {
+          "path": "nxv:endpoint",
+          "name": "endpoint",
+          "description": "The remote nexus endpoint.",
+          "nodeKind": "sh:IRI",
+          "minCount": 1
```
Comment (bogdanromanx) [inline src/main/resources/schemas/view.json]: also maxCount

File: src/main/resources/schemas/view.json [hunk 3]
```diff
@@ -234,6 +237,82 @@
...
+          "name": "Include deprecated",
+          "description": "flag to include or exclude deprecated resources on the indexed Document",
+          "datatype": "xsd:boolean"
+        },
+        {
+          "path": "nxv:project",
+          "name": "project label",
+          "description": "The project label.",
+          "datatype": "xsd:string",
+          "minCount": 1
```
Comment (bogdanromanx) [inline src/main/resources/schemas/view.json]: also maxCount

File: src/main/resources/schemas/view.json [hunk 4]
```diff
@@ -234,6 +237,82 @@
...
+        {
+          "path": "nxv:includeMetadata",
+          "name": "Include metadata",
+          "description": "flag to include or exclude metadata on the indexed Document",
+          "datatype": "xsd:boolean"
+        },
+        {
+          "path": "nxv:includeDeprecated",
+          "name": "Include deprecated",
+          "description": "flag to include or exclude deprecated resources on the indexed Document",
```
Comment (bogdanromanx) [inline src/main/resources/schemas/view.json]: drop
> on the indexed Document

File: src/main/resources/schemas/view.json [hunk 5]
```diff
@@ -234,6 +237,82 @@
...
+        {
+          "path": "nxv:resourceTag",
+          "name": "Resource tag",
+          "description": "The tag for which indexing will be triggered",
+          "datatype": "xsd:string"
+        },
+        {
+          "path": "nxv:includeMetadata",
+          "name": "Include metadata",
+          "description": "flag to include or exclude metadata on the indexed Document",
```
Comment (bogdanromanx) [inline src/main/resources/schemas/view.json]: drop
> on the indexed Document

File: src/main/scala/ch/epfl/bluebrain/nexus/kg/async/ProjectViewCoordinatorActor.scala [hunk 6]
```diff
@@ -133,24 +135,49 @@ private abstract class ProjectViewCoordinatorActor(viewCache: ViewCache[Task])(
...
+    implicit val indexing: IndexingConfig = config.elasticSearch.indexing
+    ProgressFlowElem[Task, Any]
+      .collectCast[Event]
+      .groupedWithin(indexing.batch, indexing.batchTimeout)
+      .distinct()
+      .mergeEmit()
+      .toProgress(initial)
+  }
+
+  private def projectStreamId(): String = s"project-event-count-${UUID.randomUUID()}"
```
Comment (bogdanromanx) [inline src/main/scala/ch/epfl/bluebrain/nexus/kg/async/ProjectViewCoordinatorActor.scala]: what does "count" mean in this context?

File: src/main/resources/schemas/view.json (hunk 1 above)
Comment (umbreak) [reply src/main/resources/schemas/view.json]: mmm...so far yes, the metadata could be included in the resource payload or not.

File: src/main/scala/ch/epfl/bluebrain/nexus/kg/async/ProjectViewCoordinatorActor.scala (hunk 6 above)
Comment (umbreak) [reply src/main/scala/ch/epfl/bluebrain/nexus/kg/async/ProjectViewCoordinatorActor.scala]: I removed the `count` from the word

**Pull request merged**